src/libslic3r/PrintConfig.cpp

```diff
@@ -2495,11 +2495,11 @@ def = this->add("filament_loading_speed", coFloats);
             (void)L("Maximum speed E");
             def->category = L("Machine limits");
             def->readonly = false;
-            def->tooltip  = (boost::format("Maximum speed of %1% axis") % axis_upper).str();
-            (void)L("Maximum X speed");
-            (void)L("Maximum Y speed");
-            (void)L("Maximum Z speed");
-            (void)L("Maximum E speed");
+            def->tooltip  = (boost::format("Maximum speed of the %1% axis") % axis_upper).str();
+            (void)L("Maximum speed of the X axis");
```

Comment:
The original text looks fine to me.
Can we revert the changes so that we won't loose all the translations?